src/align_navigation/scripts/goal_publisher.py

```diff
@@ -236,37 +311,44 @@ def docking_execution():
             target_distance = math.sqrt(dx*dx + dy*dy)
             diff_angle = goal[2] - pix_bot_theta
             print("Pose_Error (Estimated) (cm,degree): = ", target_distance*100, diff_angle * 180/np.pi)
-            # if dock_error < 0.05:
-            #     print("Dock_Error (Measured) (cm): = ", dock_error*100)
+            if dock_error < 0.05:
+                print("Dock_Error (Measured) (cm): = ", dock_error*100)
             dock_error = 0
             # print(state)
             StateUpdateMsg.TransState = StateOut.State_Verify
             if target_distance > 0.1 or dock_error > 0.20:
```

Comment:
Q: does it not retrace if pose error is high?

Reply:
That's there on line 319, this is just a log. Retrace looks at both errors dock and target pose

Reply:
Actually I am setting dock error as 0, so that only pose error is considered. Let me fix that